src/components/external-source/ExternalSourcePanelEntry.svelte

```diff
@@ -0,0 +1,140 @@
+  }
+</script>
+
+<div class="external-source-pairing">
+  <Collapse
+    title={derivationGroup.name}
+    tooltipContent={'Derivation group ' + derivationGroup.name}
+    defaultExpanded={false}
+  >
+    <span slot="right" style:display="flex">
+      <p class="derived-event-text">
+        {derivationGroup.derived_event_total} derived events
+      </p>
+      {#if enabled === true}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Show in timeline', placement: 'top' }}
+        >
+          <Eye style="color: rgba(143, 143, 143, 1)" />
+        </button>
+      {:else}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Hide in timeline', placement: 'top' }}
+        >
+          <EyeSlash style="color: rgba(143, 143, 143, 0.5)" />
+        </button>
+      {/if}
+    </span>
+
+    {#if relevantSources.length}
+      {#each relevantSources as source}
+        <!-- Collapsible details -->
+        <Collapse title={source.pkey.key} tooltipContent={source.pkey.key} defaultExpanded={false}>
+          <span slot="right">
+            <p style:color="gray" style:text-wrap="nowrap">
+              {derivationGroup.sources.get(source.pkey.key)?.event_counts} events
+            </p>
+          </span>
+          <p>
+            <strong>Key:</strong>
+            {source.pkey.key}
+          </p>
+
+          <p>
+            <strong>Source Type:</strong>
+            {source.source_type_name}
+          </p>
+
+          <p>
+            <strong>Start Time:</strong>
+            {formatDate(new Date(source.start_time), $plugins.time.primary.format)}
+          </p>
+
+          <p>
+            <strong>End Time:</strong>
+            {formatDate(new Date(source.end_time), $plugins.time.primary.format)}
+          </p>
+
+          <p>
+            <strong>Valid At:</strong>
+            {formatDate(new Date(source.valid_at), $plugins.time.primary.format)}
+          </p>
+
+          <p>
+            <strong>Created At:</strong>
+            {formatDate(new Date(source.created_at), $plugins.time.primary.format)}
+          </p>
+        </Collapse>
+      {/each}
+      <Collapse
+        className="anchor-collapse"
+        defaultExpanded={false}
+        title="Event Types"
+        tooltipContent="View Contained Event Types"
+      >
+        {#each derivationGroup.event_types as eventType}
+          <i>{eventType}</i>
+        {/each}
+      </Collapse>
+    {:else}
+      <p>No sources in this group.</p>
+      <button name="delete-dg" class="st-button secondary" on:click|stopPropagation={deleteEmptyDerivationGroup}>
+        Delete Empty Derivation Group
+      </button>
+    {/if}
+  </Collapse>
+</div>
+
+<style>
+  .derived-event-text {
+    align-items: center;
+    color: gray;
+    display: flex;
+    float: left;
```

Comment:
What is the purpose of `float: left` here?

Reply:
Weird - looks like this isn't replicating itself anymore for me! Removing this `float: left`.